package.json

```diff
@@ -68,6 +68,7 @@
     "proxy-polyfill": "0.1.6",
     "re-reselect": "^4.0.0",
     "react-beautiful-dnd": "^13.1.0",
+    "react-color": "^2.19.3",
     "react-datetime-picker": "^3.4.3",
```

Comment:
### Tags
- Minor: tags touching the avatar (padding?)
- Subjective:  Feels a bit odd that tag fonts are larger than channel names and "1 month ago".  Tags would be lower importance than those.

![image](https://user-images.githubusercontent.com/64950861/150708910-57338cd3-2e75-4af9-875f-7003f41e4def.png)

Reply:
I'm changing the list claim layout to something that looks more like this across all devices/resolutions. Having the icon on the same row as the meta data saves a lot of space. It's not as easy as expected tho and might take me a little bit longer to make it work in all cases.
![image](https://user-images.githubusercontent.com/94839278/150764621-a9453192-e299-497e-bc74-f8fa4143083e.png)

Reply:
Fixed. Desktop, Tablet & Mobile. The channel name and the tags have the same font size (--font-xsmall). The channel avatar is still linking to the video instead of the channel. It's on my list.
![image](https://user-images.githubusercontent.com/94839278/150772049-5aa36d6c-1ac3-43bf-a72f-5ff3b7a8f685.png)
![image](https://user-images.githubusercontent.com/94839278/150772181-fb40011f-a03f-4424-9057-60ae5ac3c8d0.png)
![image](https://user-images.githubusercontent.com/94839278/150771974-1cbeab75-f288-49bf-9354-c6a57ca33dac.png)

Reply:
This class is used in way to many unrelated places. Like the recommendations or the channel preview on the filepage itself. I'm aware of all the issues that it's causing.

Reply:
**Edit 2:** Fixed the channel avatar links. Everything should be fine now, but I'm still testing it on multiple devices.